README.md

```diff
@@ -0,0 +1,85 @@
+# Django Lightweight Queue
+
+DLQ is a lightweight & modular queue and cron system for Django.
+
+## Basic Usage
+
+``` python
+import time
+from django_lightweight_queue import task
+
+# Define a task
+@task()
+def long_running_task(first_arg, second_arg):
+    time.sleep(first_arg * second_arg)
+
+# Request that the task be executed at some point
+long_running_task(4, second_arg=9)
+```
+
+See the docstring on the [`task`](django_lightweight_queue/task.py) decorator
+for more details.
+
+## Configuration
+
+Configuration options should be placed in the usual Django settings files,
+prefixed with `LIGHTWEIGHT_QUEUE_`, for example:
```

Comment:
I'm not sure I agree -- if you put configuration options named simply `WORKERS` in the django settings files then it will also completely not work. You need to get both of these correct for it to work.

I think ideally we'd document somewhere what the options are, at which point we'd be explicit about the full names. At the moment we're relying on finding the `app_settings.py` file on their own.